src/domain/proof-of-liabilities/generate-liabilities-tree.ts

```diff
@@ -0,0 +1,33 @@
+import { getHash } from "@domain/proof-of-liabilities/generate-leaves"
+import { toSats } from "@domain/bitcoin"
+
+const getSum = (
+  val1: CurrencyBaseAmount,
+  val2: CurrencyBaseAmount,
+): CurrencyBaseAmount => {
+  return toSats(parseInt(val1.toString()) + parseInt(val2.toString()))
+}
+export const GenerateLiabilitiesTree = (leaves: TreeNode[]) => {
+  const tree = [leaves]
```

Comment:
leaves[] ?

Reply:
To construct the tree we need to first generate leaves that contain {hash , sum } and then from these leaves we construct the merkle tree.